src/ThreeEditor/util/Beam.ts

```diff
@@ -13,7 +13,14 @@ export const SIGMA_TYPE = {
 	'Flat square': 'Flat square',
 	'Flat circular': 'Flat circular'
 } as const;
-export type SigmaType = keyof typeof SIGMA_TYPE;
+export type SigmaType = keyof typeof SIGMA_TYPE
+
+export const SAD_TYPE = {
+	'None' : 'None',
+	'Rectangular': 'Rectangular',
+	'Square': 'Square'
```

Comment:
SAD dropdown menu should show:
'none',  'single (X=Y)',  'double (X and Y)'

Reply:
Please check it is OK now.